src/index.json

```diff
@@ -2582,9 +2582,9 @@
                     "metadata_version": "2.0",
                     "name": "vm-repair",
                     "summary": "Auto repair commands to fix VMs.",
-                    "version": "0.2.3"
+                    "version": "0.2.4"
                 },
-                "sha256Digest": "1dadfc622001eab35a7f8e0ecd8c02b75808e1db823fdc44b1ee5c02e9bbdf10"
```

Comment:
Is the bug fix for os_image_reference.version None case in this version of  vm_repair?

Reply:
yes. all the changes within repair_utils.py